src/explorer/Components/ExplorationInfo.cs

```diff
@@ -18,34 +16,13 @@ public ExplorationInfo(ExplorerContext ctx)
 
         public async IAsyncEnumerable<ExploreMetric> YieldMetrics()
         {
-            var r = await ResultAsync;
             yield return new UntypedMetric("exploration_info", new
             {
-                r.DataSource,
-                r.Table,
-                r.Column,
+                ctx.DataSource,
+                ctx.Table,
+                ctx.Column,
+                ctx.ColumnType,
```

Comment:
`ColumnType` is handled as an `int`. It would be more useful to handle it as a `string`.

Reply:
It's an enum and we could use the EnumConverter defined in the aircloak project to encode the return type as a string. I'll submit a patch for this.

Reply:
Done. I also moved the EnumConverter to the Diffix project, I feel this makes sense since it's where the datatype is defined, and this way the EnumConverter can be reused more easily.